pages/common/rlwrap.md

```diff
@@ -0,0 +1,24 @@
+# rlwrap
+
+> Add line editing, persistent history and prompt completion to an REPL command.
+> More information: <https://github.com/hanslub42/rlwrap>.
+
+- Run an REPL command with line editing, persistent history and prompt completion:
+
+`rlwrap {{command}}`
+
+- Use all words seen on input and output for prompt completion:
+
+`rlwrap --remember {{command}}`
+
+- Better prompt completion if prompts contain ANSI colour codes:
+
+`rlwrap --ansi-colour-aware {{command}}`
+
+- Enable filename completion (case sensitive):
+
+`rlwrap --complete-filenames {{command}}`
+
+- Add coloured prompts, use colour name, or an ASCI-conformant colour spec. Use uppercase colour name for bold styling:
```

Comment:
```suggestion
- Add coloured prompts, use colour name, or an ASCI-conformant colour specification. Use an uppercase colour name for bold styling:
```
We generally prefer using full words in descriptions (unless the line crosses the 250 characters limit).